CONTRIBUTING.md

```diff
@@ -138,3 +138,9 @@ Performing a release requires push permissions to the repository.
   - Inspect the npm registry to see the new sub packages versions.
   - Inspect the new github release named after the new `/v\d+.\d+.\d+/` tag
     and verify it contains the `.vsix` artifact.
+
+## Contributing with AI-generated code
+
+As artificial intelligence evolves, AI-generated code is becoming valuable for many software projects, including open-source initiatives. While we recognize the potential benefits of incorporating AI-generated content into our open-source projects there a certain requirements that need to be reflected and adhered to when making contributions.
+
+Please see our [guideline for AI-generated code contributions to SAP Open Source Software Projects](CONTRIBUTING_USING_GENAI.md) for these requirements.
```

Comment:
```suggestion
Please see our [guideline for AI-generated code contributions to SAP Open Source Software Projects](https://github.com/SAP/.github/blob/main/CONTRIBUTING_USING_GENAI.md) for these requirements.
```
Hi all, I'm a member of the SAP Open Source Program Office. It is ok if you create a copy of `CONTRIBUTING_USING_GENAI.md` into your repository, but unless you want to adjust it, I suggest linking to the file with have in the `.github` repo on this org, this way, if we update the guidelines centrally, you will not need to adjust it.

Reply:
@dellagustin-sap thanks. Address https://github.com/SAP/ui5-language-assistant/pull/731/commits/f75f97807e93671b56c5ed6e66f451b5431eddef